pkg/apis/kops/validation/legacy.go

```diff
@@ -157,6 +157,20 @@ func ValidateCluster(c *kops.Cluster, strict bool) *field.Error {
 		}
 	}
 
+	// Check AdditionalNetworkCIDRs
+	var additionalNetworkCIDRs []*net.IPNet
```

Comment:
This should probably be in validation.go

Reply:
Follow up PR when @justinsb can educate us.

Reply:
Added a check in validation.go but still have to leave this here as not all the check have been migrated to validation.go yet. The one that we specifically want this for is the CIDRs consistency check bellow that ensure the subnets are indeed part of the the CIDRs listed.

Reply:
Using the "new" validation functions in validation.go is probably better - the long term goal is to get all validation using those functions.  But this is fine as-is - it makes sense to keep all the network cidr validation together and move it en-masse